test/RibbonThetaVault.ts

```diff
@@ -229,6 +241,70 @@ describe("RibbonThetaVault", () => {
     },
     availableChains: [CHAINID.ETH_MAINNET],
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon NEAR Theta Vault (Call)`,
+    tokenName: "Ribbon NEAR Theta Vault",
+    tokenSymbol: "NEAR-THETA",
+    asset: NEAR_ADDRESS[chainId],
+    assetContractName: "IWBTC",
+    strikeAsset: USDC_ADDRESS[chainId],
+    collateralAsset: NEAR_ADDRESS[chainId],
+    chainlinkPricer: CHAINLINK_NEAR_PRICER[chainId],
+    deltaFirstOption: BigNumber.from("1000"),
+    deltaSecondOption: BigNumber.from("1000"),
+    deltaStep: BigNumber.from(chainId === CHAINID.ETH_MAINNET ? "100" : "10"),
```

Comment:
same

Reply:
Yeah I agree, maybe $5 for the options chain